torch/distributed/optim/zero_redundancy_optimizer.py

```diff
@@ -125,9 +125,9 @@ def __init__(
         super().__init__(params, default)
 
         # Partition information. lazy evaluation, computed if requested
-        self._per_device_params: OrderedDict[
-            torch.device, List[List[Parameter]]
-        ] = OrderedDict()  # device, rank, params
+        self._per_device_params: "OrderedDict[torch.device, List[List[Parameter]]]" = (
```

Comment:
curious, why do we using quote to wrap `OrderedDict`? And what's wrong with the previous version?

Reply:
ah that was just a tip from the IDE, that some versions of python would not support the [typeA, typeB] annotations so that it would be safer to put that into quotes ? (pylance) I can remove that

Reply:
>  that some versions of python would not support the [typeA, typeB] annotations for the OrderedDict type

I see. If Python 3.6 supports that we can safely remove as that's the earliest version that PyTorch claims to work with. Otherwise, let's keep the quotes. Thanks for explaining!

Reply:
Looks like 3.6.0 is not clear, maybe 3.6.1 https://stackoverflow.com/questions/41207128/how-do-i-specify-ordereddict-k-v-types-for-mypy-type-annotation#43583996